jena-geosparql/src/main/java/org/apache/jena/geosparql/InitGeoSPARQL.java

```diff
@@ -42,7 +42,7 @@ public static void init() {
         if ( initialized )
             return ;
         synchronized (initLock) {
-            if ( initialized ) {
+            if ( initialized || System.getProperty("jena.geosparql.skip", "false").equalsIgnoreCase("true") ) {
```

Comment:
That's going to need documenting!

Reply:
@LorenzBuehmann Do you recall the intention of this line?

One reason I can think of is that that jena-geosparql takes long to initialize. IIRC due to setting up a derby database with SIS data. So any command line jar that uses jena-geosparql takes long to load.
The loading time of own java cli tool increased from 200ms to 1.000ms just by adding jena-geosparql.

So a future issue is to change parts of jena-geosparql's eager init to lazy init on first use.